tests/WebApp.IntegrationTests/Pages/QuestionsPageTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Codidact.Core.WebApp.IntegrationTests.Pages
+{
+    public class QuestionsPageTests :
+      IClassFixture<CustomWebApplicationFactory<Startup>>
+    {
+        private readonly CustomWebApplicationFactory<Startup> _factory;
+
+        public QuestionsPageTests(
+            CustomWebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+        }
+
+        [Theory]
+        [InlineData("/questions")]
+        [InlineData("/questions?skip=0&take=20")]
+        public async Task ReturnsQuestionsByQueryParams(string url)
+        {
+            // Arrange
+            var client = _factory.CreateClient();
+
+            // Act
+            var response = await client.GetAsync(url);
+
+            // Assert
+            response.EnsureSuccessStatusCode(); // Status Code 200-299
+            Assert.Equal("text/html; charset=utf-8",
+                response.Content.Headers.ContentType.ToString());
```

Comment:
This is a bad check since it would not be incorrect if additional parameters were encoded. `response.ContentType` should be validated instead.

Reply:
I cant seem to find `response.ContentType` (mind you response is of type `HttpResponseMessage`)
Also, what do you mean additional parameters?
What additional parameters can you include on the header of ContentType?

Reply:
> I can't seem to find `response.ContentType` (mind you the response is of type HttpResponseMessage).

I was a bit sloppy there, the `response.ContentType` worked in controllers I think. `response.Content.Headers.ContentType.MediaType` should work though.

  > What additional parameters can you include on the header of ContentType?

You'd probably never see others in the field, but as far as I know it is allowed to add additional parameters.

Reply:
90c3353